assets/scripts/embedding_tests_dev_0.0.1.min.html

```diff
@@ -39,3 +39,34 @@ <h3>ERROR: 404 error</h3>
   <h3>ERROR: 404 error with deferred loading</h3>
   <oppia oppia-id="fake_id" autoload="false"></oppia>
 </div>
+
+<script type="text/javascript">
```

Comment:
Wow! This is kind of hacky, but I appreciate the difficulty.

Here's a thought for how to handle this better: add a form at the top of the page for the user to enter the exploration ID to test. When the user clicks submit, all the rest of the content on the test page loads (with the correct exp IDs). NB: if you want to make these test pages into small Angular pages, that is fine too, but keeping them as JS/jQuery is totally OK as well.

What do you think?

Reply:
Ditto below.

Reply:
nice idea!! i will add a form.
i think we should keep them to non-angular pages as embedding pages would not necessarily be angular pages.

Reply:
OK, sgtm. Thanks! (That's also nice because manual testers can also input the exploration ID manually.)

And yep, non-angular pages sgtm too.

Reply:
@seanlip i have added forms and used jquery, can you please review this.